src/internal/transactionenv/env.go

```diff
@@ -266,42 +276,21 @@ func (t *appendTransaction) DeleteRoleBinding(original *auth.Resource) error {
 // transaction is present in the RPC context.  If an active transaction is
 // present, any calls into the Transaction are first dry-run then appended
 // to the transaction.  If there is no active transaction, the request will be
-// run directly through the selected server.  A second callback may be provided
-// to override the generated transaction ID in the case that an existing
-// transaction is not being used.
-func (env *TransactionEnv) WithTransaction(ctx context.Context, cb func(Transaction) error, overrideID func(*txncontext.TransactionContext) (string, error)) error {
+// run directly through the selected server.
+func (env *TransactionEnv) WithTransaction(ctx context.Context, cb func(Transaction) error) error {
```

Comment:
Is eliminating the second callback part of the provenance change?

Reply:
It is, I see [below](https://github.com/pachyderm/pachyderm/pull/8306/files#diff-2a6378c565d5639713434161b3fcfa4246ff7e159ab117feee78a9c9d7d4ed9eL309-L327).